app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionDetailFragment.java

```diff
@@ -64,14 +65,35 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         AppUtil.setTaskDescription(activity, session.title, ContextCompat.getColor(activity, session.category.getVividColorResId()));
     }
 
+    // Receive all viewModel's properties changed.
+    private final Observable.OnPropertyChangedCallback onPropertyChanged = new Observable.OnPropertyChangedCallback() {
```

Comment:
In origin, the problem is that I have to call `setResult()` method is SessionDetailFragment. This is because I want to refresh the session checked state in sessions list (Previous Activity).

If I does not use EventBus, I have to call `setResult()` and catch values in `onActivityResult()`. Do you think it's better way to notify model changed to previous activity by using `startActivityResult()` and  `onActivityResult()`? 

Sorry for long discussion 🙇